### PR TITLE
Upgrade to Spring Boot 2.7.3 and Gradle 7.5.1

### PR DESCRIPTION
**Pull request by dgcd**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -3,12 +3,12 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.springframework.samples</groupId>
   <artifactId>spring-petclinic</artifactId>
-  <version>2.7.0-SNAPSHOT</version>
+  <version>2.7.3-SNAPSHOT</version>
```
Comment (dsyer) [inline pom.xml:6]: 2.7.3 is released, right? So no need for snapshots.

File: build.gradle [hunk 2]
```diff
@@ -36,6 +36,10 @@ dependencies {
   testImplementation 'org.springframework.boot:spring-boot-starter-test'
 }
 
-test {
+tasks.named('test') {
   useJUnitPlatform()
 }
+
+tasks.named('jar') {
```
Comment (dsyer) [inline build.gradle:43]: I don't see this in start.spring.io. What's the rrationale?

**Review by dsyer — COMMENTED** [review]
Thanks. Some comments inline.

File: pom.xml (hunk 1 above)
Comment (dgcd) [reply pom.xml]: Fixed.

File: build.gradle (hunk 2 above)
Comment (dgcd) [reply build.gradle:43]: By default it generates plain-jar alongside uber-jar.
It is not a problem, just unnecessary task and file.

**Comment by dsyer** [comment]
Merged as 276880e. Thanks.

**Pull request closed without merging**